test/recipe-test.js

```diff
@@ -0,0 +1,96 @@
+import { expect } from 'chai';
+import Recipe from '../src/classes/recipe';
+
+describe('Recipe', () => {
+    let recipe;
+    let ingredient1;
+    let ingredient2;
+    let ingredientDataSet;
+    let recipeDataSet;
+    let recipeDetails;
+
+  beforeEach(() => {
```

Comment:
This is great! Josh made a recipe data file that we can use here instead, later on during a refactor :)